cmd/stacker/build.go

```diff
@@ -52,12 +52,12 @@ func initCommonBuildFlags() []cli.Flag {
 		},
 		&cli.StringSliceFlag{
 			Name:  "layer-type",
-			Usage: "set the output layer type (supported values: tar, squashfs); can be supplied multiple times",
+			Usage: "set the output layer type (supported values: tar, squashfs, erofs); can be supplied multiple times",
 			Value: cli.NewStringSlice("tar"),
 		},
 		&cli.BoolFlag{
-			Name:  "no-squashfs-verity",
```

Comment:
I think we need an cli alias here since users of stacker may be passing `--no-squashfs-verity`